### PR TITLE
Update dotnet to 2.2.0

### PR DESCRIPTION
**Pull request by iamjoeker**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

File: Casks/dotnet.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'dotnet' do
-  version '2.1.5'
-  sha256 '4475470a353233f50ea83775f86cfcec5c58d98c82cb3222600b44eb934bb656'
+  version '2.2.0'
+  sha256 'ec993975d26e907f5d145206aac3cf078b20625e038f3821a8fb13af28ff001f' # DevSkim: ignore DS173237
 
-  url "https://download.visualstudio.microsoft.com/download/pr/b83df670-529a-4d28-b4f5-81c46d339ef9/6d23d425c2b0ffd9828c521dbf19ef63/dotnet-runtime-#{version}-osx-x64.pkg"
+  url "https://download.visualstudio.microsoft.com/download/pr/1d8682c5-f211-4724-9f6b-0d44eb1593d4/32d8ff5607122a05e2200fe0961cfc40/dotnet-runtime-#{version}-osx-x64.tar.gz"
```
Comment (ran-dall) [inline Casks/dotnet.rb]:
```ruby
https://download.visualstudio.microsoft.com/download/pr/953c69dc-2b70-4237-89e8-d0675d8e89b7/4ccde130c14f69659da5826c0b2fbe95/dotnet-runtime-#{version}-osx-x64.pkg
```

I'm pretty sure that's the right link; you have the link to the **Binaries** not the **Installer**. 👌

Comment (ran-dall) [inline Casks/dotnet.rb]:
```ruby
https://download.visualstudio.microsoft.com/download/pr/953c69dc-2b70-4237-89e8-d0675d8e89b7/4ccde130c14f69659da5826c0b2fbe95/dotnet-runtime-#{version}-osx-x64.pkg
```
> https://dotnet.microsoft.com/download/dotnet-core/2.2

**Review by ran-dall — CHANGES_REQUESTED** [review]
I'm pretty sure that's the right link; you have the link to the **Binaries** not the **Installer**. 👌

File: Casks/dotnet.rb (hunk 1 above)
Comment (iamjoeker) [reply Casks/dotnet.rb]: Right you are. Sorry about that! I push the correct url shortly.

**Pull request merged**